packages/expo-modules-autolinking/src/autolinking/findModules.ts

```diff
@@ -128,9 +128,20 @@ function addRevisionToResults(
  */
 async function findPackagesConfigPathsAsync(searchPath: string): Promise<string[]> {
   const bracedFilenames = '{' + EXPO_MODULE_CONFIG_FILENAMES.join(',') + '}';
-  const paths = await glob([`*/${bracedFilenames}`, `@*/*/${bracedFilenames}`], {
-    cwd: searchPath,
-  });
+  const paths = await glob(
+    [
+      `*/${bracedFilenames}`,
+      `@*/*/${bracedFilenames}`,
+
+      // Search config inside additional `expo` directory.
+      // This is for expo adapter in third party modules.
+      `*/expo/${bracedFilenames}`,
+      `@*/*/expo/${bracedFilenames}`,
```

Comment:
Is it still necessary to search for the module config in `expo` folder? That's also something I would like to avoid by having `expo-module.config.json` only in the root folder of the package.